css/style.css

```diff
@@ -109,3 +111,181 @@ html {
     margin-left: 1rem;
   }
 }
+.hug {
+  text-align: center;
+}
+.hug__img {
+  height: 40px;
+  width: 40px;
+  left: 144px;
+  top: 4px;
+  border-radius: 32px;
+  padding: 8px;
+}
+
+.recent__work {
+  text-align: center;
+}
+.recent__work--text {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0em;
+  Fill: Solid;
+  color: #172B4D;
+  box-sizing: border-box;
+  height: 97px;
+  width: 327px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  justify-content: center;
+  grid-gap: 16px;
+}
+
+.work {
+  border-radius: 8px;
+  margin: 20px;
+  border: 1px solid #D0D9D4;
+  height: 100%;
+}
+.work__img {
+  width: 100%;
+  height: 60%;
+  border-radius: 8px;
+}
```

Comment:
- After addressing the previous issue, remove the height of 60% to make your image have its original height and to make it appear as it should be to improve its quality.